src/ui/components/Transcript/Transcript.tsx

```diff
@@ -19,15 +18,13 @@ export default function Transcript() {
   const { isAuthenticated } = useAuth0();
 
   const displayedComments = useMemo(() => {
-    const displayedComments: Comment[] = [...comments].filter(
-      comment => !pendingComment || commentKey(comment) != commentKey(pendingComment.comment)
-    );
```

Comment:
We don't need to do this dedupe pass anymore since we removed the optimistic cache update for new comments.